### ISSUE TITLE
CTC Segmentation for German 

### ISSUE THREAD

**Issue by sadia95** [lumaku/ctc-segmentation#14]
Hello, I have to split the audio files in my dataset and their corresponding transcripts as well. Is there any pretrained model of yours for German language?

**Comment by lumaku** [comment]
I haven't found the time to prepare models for publication.....
I currently only have an older espnet 1 Transformer-based model for German (from the paper). This model is not compatible with espnet 2, which would it make it easier for you. This still needs a corpus-like setup.
I think it might be easier for you to start with the example Wav2vec2 model which is also with a German model, see https://github.com/lumaku/ctc-segmentation#usage

**Comment by lumaku** [comment]
@sadia95 And, what model/toolkit did you choose?

**Comment by lumaku** [comment]
I have not tested Nemo before. Do you use the "STT De Quartznet15x5" model? Have you tested the ASR performance on your data? This heavily influences accuracy, because alignments can only be as accurate as its CTC output. (I had good alignments with the Wav2vec2 model....)

Also, which version of the ctc-segmentation Package do you use?
Ther last versions brought some improvements. For example, text preparation with already tokenized text - I recommend to use this.
Also, I always set this option to True to handle noisy audio:
https://github.com/lumaku/ctc-segmentation/blob/master/ctc_segmentation/ctc_segmentation.py#L53

Make sure that your input data fits to the network - in your case, only lower case characters. A very useful tip is to convert numbers to their spoken equivalent. If you don't, the CTC segmentation will just filter out any utterances with numbers.

**Comment by sadia95** [comment]
Yes I am using the "STT De Quartznet15x5" model. 
Yes, I tested ASR on some of my data which I segmented by CTC-segmentation. But I haven't trained it on all of my dataset yet. 

I am using ctc-segmentation==1.1.0 version. Is that the latest one? 
Oh I also need to set it to True, because my audio files are very noisy. Ah, and I haven't been converting the numbers to alphabets either. Will do that. 
I will again look into Wav2vec model to segment my dataset, if you got good alignments with it, should also work for me.  

Thank you very much for your detailed answer. May I reach out again if need some suggestions?

**Comment by lumaku** [comment]
Currently on Version [![version](https://img.shields.io/pypi/v/ctc-segmentation?style=plastic)](https://pypi.org/project/ctc-segmentation/). However, the latest version will not work with NeMo, as the interface changed, rather use version 1.4.0 for NeMo.
If you have more questions, just write me an email!
